src/Integration.Vsix/SonarLintIntegrationPackage.cs

```diff
@@ -88,7 +87,7 @@ private async System.Threading.Tasks.Task InitOnUIThreadAsync()
                 IServiceProvider serviceProvider = this;
 
                 var activeSolutionBoundTracker = await this.GetMefServiceAsync<IActiveSolutionBoundTracker>();
-                var sonarQubeIssuesProvider = await this.GetMefServiceAsync<ISonarQubeIssuesProvider>();
+                var sonarQubeIssuesProvider = await this.GetMefServiceAsync<Core.ISonarQubeIssuesProvider>();
```

Comment:
I'm qualifying the type name with `Core` here and in a few other places rather than adding a `using` statement because otherwise there is a naming conflict (both `Core` and one of the VS assemblies have an `ErrorHandler` class).